### DISCUSSION TITLE
如何接入gitlab搭建自己的线上webide

### DISCUSSION THREAD

**Discussion by zhubin1992** [opensumi/core#2629] [Q&A]
官网的ide-startup例子只能使用本地资源。要达成标题的目的，总不能每次都拉取代码，我需要接入gitlab的openApi么，有没有示例

**Comment by erha19** [comment]
@zhubin1992  看起来你更需要的是纯前端的 IDE 能力，https://github.com/opensumi/ide-startup-lite 这个例子接入了 GitHub 的 OpenAPI ，逻辑是类似的，也可以在 WebIDE 版本中实现相似能力
